pkgs/tools/inputmethods/keym/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libX11
+, libXtst
+, libXi
+}:
+
+stdenv.mkDerivation rec {
+  pname = "keym";
+  version = "unstable-2022-07-10";
+
+  src = fetchFromGitHub {
+    owner = "cwkx";
+    repo = pname;
```

```suggestion
    repo = "keym";
```

pname and repo are semantically different